wind_path: /home/jakeware/Dropbox/MIT/rrg/flow_field/fluent/exported_data/single_building_v2_comparison/steady_rsm_lps/data_it6000/
x_fname: reg_pos_x.csv
y_fname: reg_pos_y.csv
z_fname: reg_pos_z.csv
u_fname: reg_mean_vel_x.csv
v_fname: reg_mean_vel_y.csv
w_fname: reg_mean_vel_z.csv
s2uu_fname: reg_cov_vel_xx.csv
s2vv_fname: reg_cov_vel_yy.csv
s2ww_fname: reg_cov_vel_zz.csv
s2uv_fname: reg_cov_vel_xy.csv
s2vw_fname: reg_cov_vel_yz.csv
s2uw_fname: reg_cov_vel_xz.csv
obs_mask_fname: obs_mask.csv
grid_size_fname: grid_size.csv
grid_res_fname: xy_step.csv
grid_height_fname: z_samp.csv